tests/fast-integration/monitoring/prometheus.js

```diff
@@ -140,14 +140,13 @@ function shouldIgnoreTarget(target) {
 }
 
 function shouldIgnoreAlert(alert) {
+    // List of alerts that we don't care about and should be filtered
     var alertNamesToIgnore = [
-        // Watchdog is an alert meant to ensure that the entire alerting pipeline is functional, so it should always be firing,
+        // Watchdog is an alert meant to ensure that the entire alerting pipeline is functional
         "Watchdog",
         // Scrape limits can be exceeded on long-running clusters and can be ignored
         "ScrapeLimitForTargetExceeded",
-
-        // Overcommitting means that a cluster with at least 1 node taken down doesn't have enough resources to run all the pods
-        // It's fine in an e2e test scenario since the clusters are usually deliberately created small to save money
+        // If a cluster with at least 1 node taken down doesn't have enough resources to run all the pods which is fine for e2e test scenarios
```

Comment:
```suggestion
        // If a cluster with at least 1 node taken down doesn't have enough resources to run all the pods, which is fine for e2e test scenarios
```

Not sure if I'm parsing this sentence correctly. It's pretty long, but still seems to miss the bit that says what happens if a cluster (...) doesn't have enough resources.

Reply:
It's just a comment for us to know why we ignore the alert :) wanted to shorten it a bit

Reply:
Aaah of course, it follows to line 143 "alerts we don't care about".
```suggestion
        // If a cluster with at least 1 node taken down doesn't have enough resources to run all the pods; that's fine for e2e test scenarios
```

Reply:
Changed it a bit to be short and concise